modules/koblerBidAdapter.md

```diff
@@ -38,8 +38,8 @@ Kobler identifies the placement using the combination of the page URL and the al
 ```
 
 In order to see a sample bid from Kobler (without a proper setup), you have to also do the following:
-- Change the [`refererInfo` function](https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js) to return `'https://www.tv2.no/a/11734615'` as a [`referer`](https://github.com/prebid/Prebid.js/blob/caead3ccccc448e4cd09d074fd9f8833f56fe9b3/src/refererDetection.js#L169). This is necessary because Kobler only bids on recognized articles. 
-- Change the adapter's [`BIDDER_ENDPOINT`](https://github.com/prebid/Prebid.js/blob/master/modules/koblerBidAdapter.js#L8) to `'https://bid-service.dev.essrtb.com/bid/prebid_rtb_call'`. This endpoint belongs to the development server that is set up to always return a bid for the correct format size and page URL combination. 
+- Set the `test` parameter to `true`.
+- Set config.pageUrl to `'https://www.tv2.no/mening-og-analyse/14555348/'`. This is necessary because Kobler only bids on recognized articles.
```

Comment:
Maybe we should clarify what this does and explain that this is just an example URL (I guess this won't work in a year for example, since we only bid on articles that are at most 1 year old)

Reply:
Okay, I'll add more details. We can still make it work in 1 year, we just have to keep updating the `date_dt` field of the document in Solr in DEV and make sure our campaign (https://dev-app.kobler.no/campaigns/572) has enough budget and its CT matches the article.